src/notes.md

```diff
@@ -0,0 +1,51 @@
+# Intro
+- Three ways for SDK to be loaded
+  - Standard NPM package
+  - Snippet in `<script>` tag
+  - RequireJS (may be dropped in future)
+- Build system is Make instead of yarn (legacy, may possibly be updated)
+- Use Amplitude Instrumentation Explorer to help development
+  - "Instrumentation" involves matters related to logging user actions (i.e. events)
+
+# Concerns
```

Comment:
This one sounds like more useful to yourself. Only put notes which are useful for others to understand this SDK better.

Reply:
I guess we should move all this notes to a `contributing.md` file as a setup guide.